pkgs/development/python-modules/plantuml/default.nix

```diff
@@ -0,0 +1,45 @@
+{ buildPythonPackage
+, fetchFromGitHub
+, lib
+  # Runtime dependencies
+, pkgs
+, sphinx
+, httplib2
+, twine
+, wheel
+, six
+}:
+
+buildPythonPackage rec {
+  pname = "plantuml";
+  version = "0.3.0";
+
+  src = fetchFromGitHub {
+    owner = "dougn";
+    repo = "python-plantuml";
+    rev = "93e1aac25b17d896b0d05d0a1aa352c7bd11dd31";
+    sha256 = "sha256-aPXPqoKlu8VLi0Jn84brG7v3qM9L18Ut4sabYYGb3qQ=";
+  };
+
+  propagatedBuildInputs = [
+    sphinx
+    httplib2
+    twine
+    wheel
+    six
+    pkgs.plantuml
```

Comment:
Reading the code, locally installed `plantuml` is never really used, it connects to a server, so I don't think this qualifies as a dependency.